lib/rmt.rb

```diff
@@ -1,5 +1,5 @@
 module RMT
-  VERSION ||= '1.2.7'.freeze
+  VERSION ||= '1.2.8'.freeze
```

Comment:
If any users are using the CSV output as an API of sorts, outputting the headers by default would be a breaking change.

Reply:
Updated to 2.0.0

Reply:
I think it must be 2.0.0 too

Reply:
Before you merge, let me check with SLE maintainers whether it's ok to bump the major version like this,